conn-soup/README.md

```diff
@@ -134,3 +134,10 @@ networkInterface: 0.0.0.0
 # Keeps all logical message parts together. If it set to value of `true` they won't be split to different batches
 evolutionSupportEnabled: true
 ```
+
+## Release notes
+
+### 3.6.0
```

Comment:
I think we should add information about the new base image version we've started using for the conn boxes

Reply:
I will add it to the main README file, thanks